internal/pkg/describe/backend_service.go

```diff
@@ -103,19 +105,25 @@ func (d *BackendServiceDescriber) Describe() (HumanJSONStringer, error) {
 				Endpoint: endpoint,
 			}, env)
 		}
+		containerPlatform, err := d.svcStackDescriber[env].Platform()
+		if err != nil {
+			return nil, fmt.Errorf("retrieve platform: %w", err)
+		}
+		platform = dockerengine.PlatformString(containerPlatform.OperatingSystem, containerPlatform.Architecture)
+		backendSvcEnvVars, err := d.svcStackDescriber[env].EnvVars()
+		if err != nil {
+			return nil, fmt.Errorf("retrieve environment variables: %w", err)
+		}
 		configs = append(configs, &ECSServiceConfig{
 			ServiceConfig: &ServiceConfig{
 				Environment: env,
 				Port:        port,
 				CPU:         svcParams[cfnstack.WorkloadTaskCPUParamKey],
 				Memory:      svcParams[cfnstack.WorkloadTaskMemoryParamKey],
+				Platform:    platform,
```

Comment:
nit: can we remove the variable completely and have the platform inlined like this:

```suggestion
				Platform:    dockerengine.PlatformString(containerPlatform.OperatingSystem, containerPlatform.Architecture),
```

Reply:
Yup. Done.